src/aqua/installation-spell/src/aqua/spell.aqua

```diff
@@ -152,17 +166,7 @@ func install(ipfs: Multiaddr, worker_def_cid: CID):
 
             for s <- worker_definition.services:
                 try:
-                    hashes: *Hash
-                    for m <- s.modules:
-                        get_wasm <- Ipfs.get_from(m.wasm, ipfs)
-                        json_cfg <- Ipfs.cat_from(m.config, ipfs)
-                        ipfs_ok <- and(get_wasm.success, json_cfg.success)
-                        if ipfs_ok:
-                            cfg <- ModuleConfigJson.parse(json_cfg.contents)
-                            hashes <- Dist.add_module_from_vault(get_wasm.path, cfg)
-                        else:
-                            log([s.name, "error retrieving module", ["wasm", m.wasm, get_wasm], ["config", m.config, json_cfg]])
-
+                    hashes = download_service(s, ipfs)
```

Comment:
```suggestion
                    hashes <- download_service(s, ipfs)
```

Reply:
Is it okay? Do I need to rewrite everything with `hashes <- download_services()`? It would require ugly stuff like `hashes!.length` and so on